pallets/inflation/src/lib.rs

```diff
@@ -589,3 +597,42 @@ pub trait PayoutPerBlock<Imbalance> {
     /// Payout reward to the collator responsible for producing the block.
     fn collators(reward: Imbalance);
 }
+
+/// `OnRuntimeUpgrade` logic for integrating this pallet into the live network.
+#[cfg(feature = "try-runtime")]
+use sp_std::vec::Vec;
```

Comment:
Can't this be moved into 'post_upgrade'?

Reply:
Not as far as I know, it's already used in the function footprint.